streampipes-commons/src/main/java/org/apache/streampipes/commons/constants/Envs.java

```diff
@@ -23,9 +23,9 @@ public enum Envs {
   SP_PORT("SP_PORT", null),
 
   @Deprecated(since = "0.90.0", forRemoval = true)
-  SP_CONSUL_LOCATION("CONSUL_LOCATION", "consul"),
+  SP_CONSUL_LOCATION("CONSUL_LOCATION", "consul", "localhost"),
```

Comment:
Is the deprecation here still valid?

Reply:
Yes, but we can remove this in the next version